client/data/sites/use-launchpad.ts

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from 'react-query';
+import wpcom from 'calypso/lib/wp';
+
+export const fetchLaunchpad = ( siteSlug: string ) => {
+	const slug = encodeURIComponent( siteSlug as string );
+	return wpcom.req.get( {
+		path: `/sites/${ slug }/launchpad`,
+		apiNamespace: 'wpcom/v2',
+	} );
+};
+
+export const useLaunchpad = ( siteSlug: string, cache = true ) => {
```

Comment:
Do we need to worry about React Query caching this value, and not updating quickly after the setting has been updated on the backend?

Reply:
Worth noting I tested this - by completing Launchpad, which turns the launchpad_screen option to off, and then immediately navigating to My Home. Seems to work fine. If I recall, RQ may automatticaly refetch data on window refreshes. Not sure how that interacts with the meta -> persist property here, but in any case, seems to be working just fine - no stale data when arriving on My Home.

Reply:
Looking at the React Query docs, it seems like the query will refetch when its mounted again or the user refocuses on the window by default: https://react-query-v3.tanstack.com/guides/caching

For now we may be fine with the react query default values